src/tests/test_trivia.py

```diff
@@ -0,0 +1,36 @@
+import os
+import json
+import pytest
+from ..service.trivia import TriviaService
```

Comment:
Please add a newline below and remove the relative import
```suggestion
from service.trivia import TriviaService

```